<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>phyca</title>
  <link rel="stylesheet" href="./CSS/style.css">
  <link href="./CSS/phylotree.css" rel="stylesheet" />
  <!-- Latest compiled and minified CSS -->
  <script src="https://code.jquery.com/jquery.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous" />
  <link href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet" />
  <!-- Optional theme -->
  <link rel="stylesheet" href="//stackpath.bootstrapcdn.com/bootswatch/4.1.1/spacelab/bootstrap.min.css" />
  <!-- Latest compiled and minified JavaScript -->
  <script src="//cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
    integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
    crossorigin="anonymous"></script>
  <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
    integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
    crossorigin="anonymous"></script>
  <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0">-->
  <script src="//d3js.org/d3.v6.js"></script>
  <script src="./JS/phylotree.min.js?123"></script>
</head>
<body>
  <nav class="navbar navbar-expand-lg navbar-light text-white" style="background-color: #0048ff;">
    <div class="container px-1">
      <!-- Left side with logo -->
      <a class="text-white h6" href="./index.html">
        Universal Ortholog Based Phylogenomic Database and Toolset
      </a>
      <!-- Navbar toggle button for small screens -->
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
        aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <!-- Right side with links -->
      <div class="collapse navbar-collapse" id="navbarNav">
        <ul class="navbar-nav ml-auto">
          <li class="nav-item ">
            <a class="nav-link" href="./index.html">Home </a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="./genomes.html">Genomes</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="./data.html">Data</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="./software.html">Software</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="./about.html">About</a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
  <section>

    &nbsp;
    &nbsp;
    &nbsp;
    &nbsp;

    <div class="container" id="mainContainer">

      &nbsp;
      &nbsp;
      &nbsp;
      &nbsp;

    <h3><b>phyca</b>: <b>phy</b>logeny and <b>c</b>ollinearity <b>a</b>ware assembly evaluation toolkit</h3>

      &nbsp;
      &nbsp;
      <p>

        phyca is built around <a href="https://github.com/huangnengCSU/compleasm">Compleasm</a> utilizing the <a href="https://www.ncbi.nlm.nih.gov/datasets/genome/">NCBI Genome</a> database. For a query assembly, phyca improves the precision of BUSCO/Compleasm annotations by up to 7%, makes syntenic comparisons to public reference genomes and rapidly places the assembly on a broad, precomputed phylogeny.

        <br>
        <br>
        Source and documentation: <a href="https://github.com/DeadlineWasYesterday/phyca/">https://github.com/DeadlineWasYesterday/phyca/</a>
        

      </p>
      &nbsp;
      &nbsp;

    
    <h3><b>BioNick</b>: Software for manipulating newick strings and adding phylogenies to Matplotlib visuals</h3>

      &nbsp;
      &nbsp;

      <p>

      BioNick includes a series of modular functions for the manipulation of Newick strings in python, e.g., extracting leaves, swapping roots, removing node labels, flipping the order of nodes, removing leaves, extracting subtrees, visualizing cladograms with matplotlib visuals, etc. BioNick is also equipped with the ability to represent trees as a collection of node objects and create Neighbor-Joining trees from distance matrices.
      &nbsp;
        <br>
        <br>

      Source and documentation: <a href="https://github.com/DeadlineWasYesterday/BioNick/"> https://github.com/DeadlineWasYesterday/BioNick/ </a>

      </p>
      &nbsp;
      &nbsp;


    <h3><b>stax</b>: stacking tool for collapsing inflated genome assemblies [Under Development]</h3>


  </div>


  </section>
  <!-- <footer class="mt-5 text-center footer-section ">
    <h6 class="text-center  text-white m-0 "> Website Developed by <a href="https://www.eezahtech.com/" target="_blank"
        class=" footer-link "  >EezahTech.com</a> </h6>
    <h6 class="text-center  text-white m-0 ">Treeviewer adapted from <a href="https://github.com/veg/phylotree.js"
      class=" footer-link" target="_blank"> https://github.com/veg/phylotree.js</a> </h6>
  </footer> -->
  <script src="./JS/phylotreeIndexFile.js"></script>
  <script src="./JS/logic.js"></script>
</body>
</html>